code/modules/surgery/organs/organ_external.dm

```diff
@@ -487,14 +487,43 @@ Note that amputating the affected organ does in fact remove the infection from t
 		//Open wounds can become infected
 		if(owner.germ_level > W.germ_level && W.infection_check())
 			W.germ_level++
-
+	for(var/obj/item/organ/external/O in owner)
+		//incisions make it easier to get an infection
+		if(O.open)
+			spread_germs_by_incision(O)
```

Comment:
This is handling it very poorly.

`handle_germ_sync` is called *individually* on every single external organ that processes...then what you do here is have it loop through every single external organ inside this proc, itself.

I think you can see the issue with this; a right arm is going to process, which will then cause this proc to be called on all other external organs. Then the right hand will process and this cycle repeats...despite not really needing to spread germs *again*.

This should only occur once per organ.